CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## WIP
+
+- Write dump information and dump progress bar to stderr when verbose mode is enabled ([#113](https://github.com/Smile-SA/gdpr-dump/pull/113))
+- Better converter condition validation by using a php tokenizer ([#114](https://github.com/Smile-SA/gdpr-dump/pull/114))
+- Upgraded druidfi/mysqldump-php version from `^1.0` to `^1.1` ([#117](https://github.com/Smile-SA/gdpr-dump/pull/117))
```

Comment:
```suggestion
- Upgraded druidfi/mysqldump-php version from `^1.0` to `^1.1` to improve dump performance 8-10x ([#117](https://github.com/Smile-SA/gdpr-dump/pull/117))
```

Reply:
Actually it's not for that reason that I need to upgrade the min required version.
It's because the value returned by the setInfoHook has changed (new array key `completed`).

The performance improvement is awesome, but it's not a breaking change.